docs/requirements.txt

```diff
@@ -4,5 +4,5 @@ numpy>=1.7
 contextlib2
 cloudpickle>=0.3.1
 graphviz>=0.8
-networkx>=2.0.0
+networkx>=2.2rc1
```

Comment:
Sure, let's just remember to update this to `>=2.2` before the next Pyro release.